src/xcode/ENA/ENA/Source/Client/HTTP Client/__tests__/HTTPClient+SubmitTests.swift

```diff
@@ -172,7 +172,7 @@ final class HTTPClientSubmitTests: XCTestCase {
 		let expectation = self.expectation(description: "Response403")
 
 		// Act
-		let payload = CountrySubmissionPayload(exposureKeys: keys, visitedCountries: [], tan: tan)
+		let payload = CountrySubmissionPayload(exposureKeys: keys, visitedCountries: [], eventCheckIns: [], tan: tan)
 		HTTPClient.makeWith(mock: stack).submit(payload: payload, isFake: false) { response in
 			defer { expectation.fulfill() }
 
```

Comment:
Are there missing tests for not empty lists of eventCheckIns?

Reply:
The tests are work in progress. My plan to add them today was delayed but I'm on them.

Reply:
I checked the existing tests and found out a couple of issues with them:
* All tests use the `MockNetworkStack` which defines the response. Providing multiple CheckIns wouldn't change anything as there are no sanity checks for the provided data
* None of the tests provides `visitedCountries` as well. I guess for the aforementioned reason.

Generally we should consider live-server tests with 'real' payloads to test our code under real conditions. Happy to discuss this after the vacation. For now I would keep this in mind (and noted) but wouldn't extend these tests more than necessary.